plugins/console/spec/consoleConfig.js

```diff
@@ -0,0 +1,17 @@
+var env = require('../../../spec/environment.js');
```

Comment:
Thanks I was looking for that.
One quick question for you, because I don't think my tests are very good. I cant think of a way to test that the tests fail without failing the test if you get me :). Any idea if this is possible?